core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -1644,6 +1656,7 @@ class PlaintextConsumerTest extends BaseConsumerTest {
     for (_ <- 0 until numOfConsumersToAdd) {
       val newConsumer = new KafkaConsumer[Array[Byte], Array[Byte]](this.consumerConfig)
       consumerGroup += newConsumer
+      consumers += newConsumer
```

Comment:
Don't understand your comment.

IMHO, we need this to make sure all consumers are closed in `IntegrationTestHarness#tearDown` -- otherwise, their IDs are not deregistered and the metic registry is not clear between tests (it's the same JVM running multiple tests). Or do I miss something?